lib/cublas/wrappers.jl

```diff
@@ -115,7 +115,7 @@ for (fname, fname_64, elty) in ((:cublasDscal_v2, :cublasDscal_v2_64, :Float64),
                                 (:cublasCscal_v2, :cublasCscal_v2_64, :ComplexF32))
     @eval begin
         function scal!(n::Integer,
-                       alpha::Number,
+                alpha::CuRefArray{$elty, CuVector{$elty, DeviceMemory}},
                        x::StridedCuVecOrDenseMat{$elty})
```

Comment:
I know this is because the formatter only ran on the diff, but I'm no fan of partially aligned signatures.

Reply:
Grrr stupid formatter, I agree